JDStatusBarNotification/JDStatusBarStyle.h

```diff
@@ -22,6 +22,10 @@ extern NSString *const JDStatusBarStyleDefault;
 /// This style has a nearly black background with a nearly white Helvetica label.
 extern NSString *const JDStatusBarStyleDark;
 
+/// This style has a red backbround with a white Helvetica label. For iPhone X, the size of
+/// the notification status bar is minimize to show the navigation bar
+extern NSString *const JDStatusBarStyleErrorMini;
```

Comment:
Let's not add a new system style. Custom styles can be used, if someone wants iPhoneX customization.